src/context/FilterProjectsContext.jsx

```diff
@@ -34,7 +34,9 @@ export const FilterProjectsProvider = ({ children }) => {
   const [enableFollowScreen, setEnableFollowScreen] = useState(false)
   const [mapBbox, setMapBbox] = useState({})
 
-  const displayedSurveys = displayedProjects.flatMap((project) => project.records)
+  const filteredSurveys = displayedProjects
+    .filter((project) => checkedProjects.includes(project.project_id))
```

Comment:
Added the filter to only include projects that are 'checked'

Reply:
(`displayedProjects` is already filtered, but confusingly, not on projects, apparently only on other filters. I will be in this codebase too briefly to confidently make a refactor to have all the filtering logic live in the same place, but worth noting  to @tienhoah who I hear will be taking over. :) )